app/views/recipes/public_recipes.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class='shopping-list'>
+ <h1 class='public-recipes-title'>Public Recipes</h1>
+ <% @public_recipes.each do |public_recipe| %>
+   <% if can? :delete, public_recipe %>
+      <%= link_to "Delete", recipe_path(public_recipe), method: :delete, data: { confirm: "Are you sure?" } %>
+    <% end %>  
+<%= link_to recipe_path(public_recipe) do %>   
+<div class='recipe-container'>
+<section class='recipe-info'>
+   <h3 class='recipe-1'><%= public_recipe.name %></h3>
+   <p class='recipe-btn'>By:<%= public_recipe.user.name %></p>
+</section> 
+<section class='public-total'>   
+   <p class='recipe-description'>Total food Items: <%= public_recipe.total_quantity %></p>
+   <p class='recipe-description'>Total price: $<%= public_recipe.total_price %></p>
+</section> 
+</div>
+<% end %>
+<% end %>
+
```

Comment:
- [ ] [MANDATORY] Just as mentioned by the previous reviewer [here](https://github.com/ismayito/Rails-recipe-app/pull/29#discussion_r1459923243), when I navigate to the pubic recipes page, I do not see anything. This is highly expected as we were not able to see the recipes page in the first place.

![image](https://github.com/ismayito/Rails-recipe-app/assets/86318284/afa94d8a-c478-49bd-ba7f-a7f41608893e)